### ISSUE TITLE
Material 3 UI for Amaze File Manager

### ISSUE THREAD

**Issue by VishalNehra** [arpitkh96/AmazeFileManager#4196]
Material design 3 UI update for Amaze File Manager
Resources
https://www.figma.com/design/kISSjFscKDtmoqkgogQUdQ/Amaze-File-Manager

Export resources:
1. Select the correct layer on the Styles page. Layers are named for your convenience.
2. On the properties bar on the left select the Export option. 

https://drive.google.com/drive/folders/1cGveyfbDARcA3p1xLn6DeURBF1DKfTIN?usp=sharing

**Comment by VishnuSanal** [comment]
PS: each individual component (say, one screen) can be treated as a single issue. will update them soon here. btw, I vouch for using jetpack compose, what say?

**Comment by animeshdeepak** [comment]
hi @VishnuSanal @VishalNehra UI seems great,
i can start working with above figma using compose UI, if allowed.

**Comment by VishalNehra** [comment]
> hi @VishnuSanal @VishalNehra UI seems great, i can start working with above figma using compose UI, if allowed.

Sure! Feel free to contact here if you get stuck or keep updated about the progress.

**Comment by VishnuSanal** [comment]
@animeshdeepak which screen are you tackling first?

**Comment by animeshdeepak** [comment]
@VishnuSanal
i was thinking of using MVI clean architecture, 
after that we will decide which screen to start first.

**Comment by VishnuSanal** [comment]
@animeshdeepak MVI? Architecture? Isn't this issue purely related to layout? CMIIW. :)

**Comment by animeshdeepak** [comment]
yes, compose UI state management works best with MVI arch.

**Comment by animeshdeepak** [comment]
@VishnuSanal are you okay with my decision for MVI arch and compose.
One more thing, when is your next release date?

**Comment by VishnuSanal** [comment]
@animeshdeepak Hi, we have MVVM arch for now. Did you meant overhauling the Arch for the compose migration? If yes, then probably not. Our plan was to slowly migrate to compose, just like we're doing it for Java -> Kotlin. And, about the release date: ":D 🏃🏿‍♀"! We haven't decided!

**Comment by EmmanuelMess** [comment]
> PS: each individual component (say, one screen) can be treated as a single issue. will update them soon here. 

Maybe subissues, related issues should be linked.

**Comment by beedell-roke** [comment]
> [`#issue-2354584389`](https://github.com/TeamAmaze/AmazeFileManager/issues/4196#issue-2354584389)

Does this duplicate [`issues/3193`](https://github.com/TeamAmaze/AmazeFileManager/issues/3193#issue-1154335846)?

> [`#issuecomment-2653870021`](https://github.com/TeamAmaze/AmazeFileManager/issues/4196#issuecomment-2653870021)

One sub-issue is [`issues/4373`](https://github.com/TeamAmaze/AmazeFileManager/issues/4373#issue-2941101433).

**Comment by soymadip** [comment]
Hey so material 3/expressive is planned?

**Comment by VishalNehra** [comment]
> Hey so material 3/expressive is planned?

yes, there's already a [WIP PR](https://github.com/TeamAmaze/AmazeFileManager/pull/4665) I'm already working on a design overhaul, we have all the UX ready.
